src/features/movies/MovieTile/index.js

```diff
@@ -34,7 +34,7 @@ const MovieTile = (props) => {
       ></MovieTileImg>
       <MovieInfoWrapper>
         <MovieTileHeader>{props.movie.title}</MovieTileHeader>
-        <MovieTileYear>{props.movie.release_date.slice(0, 4)}</MovieTileYear>
+        <MovieTileYear>{props.movie.release_date}</MovieTileYear>
```

Comment:
usunąłem tą funkcję slice, ponieważ z jakiegoś powodu zaczęła wyrzucać błędy, w komponencie MovieTile

Reply:
A zauważyłeś w jakich przypadkach wywala błędy? Może wtedy, kiedy release_date jest nullem? To wtedy można to inaczej obejść, nie trzeba rezygnować ze slice.

Reply:
właśnie z tego co widziałem to w żadnym przypadku nie było nullem kiedy był błąd, ale racja, zrobię tam warunek w kodzie